drivers/mrf24j40/mrf24j40_netdev.c

```diff
@@ -286,14 +286,27 @@ static int _get(netdev_t *netdev, netopt_t opt, void *val, size_t max_len)
             break;
 
         case NETOPT_IS_CHANNEL_CLR:
-            if (mrf24j40_cca(dev)) {
+        {
+            int8_t rssi;
+            if (mrf24j40_cca(dev, &rssi)) {
                 *((netopt_enable_t *)val) = NETOPT_ENABLE;
             }
             else {
                 *((netopt_enable_t *)val) = NETOPT_DISABLE;
             }
             res = sizeof(netopt_enable_t);
             break;
+        }
+
+        case NETOPT_LAST_ED_LEVEL:
+            if (max_len < sizeof(int8_t)) {
+                res = -EOVERFLOW;
+            }
+            else {
+              *((int8_t *)val) = mrf24j40_get_ed_level(dev);
```

Comment:
mrf24j40_cca(dev, (int8_t *)val);

Reply:
I've made your corrections ! Thank you for your help :) (and sorry for being such a noob X)

Reply:
No worries - that's why we do code reviews ;)